### PR TITLE
Fix bugs

### LINKED ISSUE #157: Swap shift command help message wrong
image

### LINKED ISSUE #170: [PE-D] Typo in delete staff command description
-------------
Labels: `severity.Low` `type.DocumentationBug`
original: whoisjustinngo/ped#12

### LINKED ISSUE #181: [PE-D] User Guide has invalid Command
Steps to reproduce:

1. Open User's Guide

Description: There is no role called cashiers

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: RachelCheah/ped#11

### LINKED ISSUE #186: [PE-D] Unmatched error message
Steps to reproduce:

1. key in `deleteShift -n Alex Yeoh d/tuesday-2 da/2021-10-01` (copied from user guide)

Expected Result: shift does not exist

Actual Result: Invalid command format! Also there's a missing space between if and only " Ifonly one date is provided, the end date is assumed to be seven days later." and another missing space between "usedto"

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: RachelCheah/ped#16

### LINKED ISSUE #218: [PE-D] DeleteShift feature - example command does not work
Followed the example command in the guide and got an invalid command error

-------------
Labels: `type.FunctionalityBug` `severity.High`
original: jason-ng-zq99/ped#11

### LINKED ISSUE #220: [PE-D] Delete error message is inconsistent
Both commands having the right format but having invalid number input display different error message.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: gordonlzy/ped#12

### PR DESCRIPTION
**Pull request by Tetrerox**
Closes #220 
Closes #218 
Closes #186 
Closes #181 
Closes #170 
Closes #157

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SwapShiftCommand.java [hunk 1]
```diff
@@ -25,14 +25,14 @@
...
-            + "shifts are active over is optional. Date input is used to indicate the period when the shift is active."
+            + "using their names. You have to input exactly 2 names and 2 shifts. The period within which the "
+            + "shifts are active over is optional. Date input is used to indicate the period when the shift is active. "
             + SHIFT_PERIOD_PARSING_DEFAULT + "\n\n"
             + "NOTE: The staff identified using the first name is associated with the first shift and the staff "
             + "identified using the second name is associated with the second shift. Do take note of the order!\n\n"
             + "Parameters: (2 of each)\n"
             + PREFIX_DASH_NAME + " NAME\n"
-            + PREFIX_DAY_SHIFT + "monday-1"
+            + PREFIX_DAY_SHIFT + "dayOfWeek-slotNumber "
```
Comment (Gabau) [inline src/main/java/seedu/address/logic/commands/SwapShiftCommand.java:35]: Should be `DAYOFWEEK-SLOTNUMBER` as fields are all caps?

**Review by Gabau — CHANGES_REQUESTED** [review]
LGTM, one formatting error spotted.

**Review by Gabau — APPROVED** [review]
LGTM

**Review by mweeruien — APPROVED** [review]
LGTM

**Pull request merged**